mobile/src/main/java/ch/epfl/sdp/mobile/application/tts/TextToSpeechFacade.kt

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.sdp.mobile.application.tts
+
+import ch.epfl.sdp.mobile.infrastructure.tts.TextToSpeechFactory
+
+class TextToSpeechFacade(private val factory: TextToSpeechFactory) {
```

Comment:
Could we add some behaviour to our existing `SpeechFacade` instead ?

Reply:
I think we may need all of it. Namely, I am thinking of synthesizing(text), stopping and shutdown() methods

Reply:
Stopping and shutdown are methods that should not be on the facade, but rather on the TextToSpeech interface.